src/neuroconv/tools/data_transfers/_aws.py

```diff
@@ -30,3 +36,283 @@ def estimate_s3_conversion_cost(
     cost_gb_m = 0.08 / 1e3  # $0.08 / GB Month
     cost_mb_s = cost_gb_m / (1e3 * 2.628e6)  # assuming 30 day month; unsure how amazon weights shorter months?
     return cost_mb_s * total_mb_s
+
+
+def submit_aws_batch_job(
+    *,
+    job_name: str,
```

Comment:
Can we make this optional and generate one automatically?

Reply:
An internal `jobId` is automatically generated and is part of the info dict returned by this function; that is the internal one used by AWS operations

The `job_name` is a user/human readable tag related to the intent/usage/reference of that ID, so I think the user should specify it how they want

Otherwise the most we would do I think is utilize the `docker_image` as the base name and add a truncated UUID on the end to make it unique; but users would HAVE to grab the `info` dict returned to know how to find the name, which could be hard if they have lots of jobs running at once

Reply:
Explained on the meeting. Thanks.